test-e2e/jest.config.js

```diff
@@ -1,5 +1,16 @@
 module.exports = {
-  moduleFileExtensions: ['sh'],
-  testRunner: './jest-bash-runner/index.js',
-  testMatch: ['**/*-test.sh']
+  projects: [
```

Comment:
didn't know jest can do multiple runners in the same config, nice!

Reply:
We could probably even move the config out to the root and have a jest-project for each part of esy.